WooCommerce/Classes/ViewRelated/Products/Variations/ProductVariationsViewController.swift

```diff
@@ -686,11 +697,12 @@ private extension ProductVariationsViewController {
         case .noResultsPlaceholder:
             break
         case .syncing(let pageNumber):
-            if pageNumber == SyncingCoordinator.Defaults.pageFirstIndex {
+            if pageNumber == syncingCoordinator.pageFirstIndex {
```

Comment:
Maybe I don't have the full context but I made this change because I think we should use the property instead of the constant since both values could be different 😅

Reply:
Agreed - this actually looks better 👍